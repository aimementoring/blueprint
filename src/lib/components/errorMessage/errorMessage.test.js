import React from 'react';
import renderer from 'react-test-renderer';
import ErrorMessage from './errorMessage';

describe('ErrorMessage', () => {
  it('renders properly', () => {
    const tree = renderer.create(<ErrorMessage message="Testing error message" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
