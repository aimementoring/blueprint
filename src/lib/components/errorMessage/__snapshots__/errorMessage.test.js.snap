// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage renders properly 1`] = `
<div
  className="theme-base"
>
  <div
    className="textContainer active"
  >
    <span
      className="errorMessage"
    >
      Testing error message
    </span>
  </div>
</div>
`;
